apps/judicial-system/web/src/routes/PublicProsecutor/Indictments/Overview/Overview.tsx

```
@@ -96,11 +144,76 @@ export const Overview = () => {
+            workingCase.indictmentRulingDecision ===
+            CaseIndictmentRulingDecision.FINE
+
+          const serviceRequired =
+            defendant.serviceRequirement === ServiceRequirement.REQUIRED
+
+          return (
+            <>
+              <Box component="section" marginBottom={3} key={defendant.id}>
+                <BlueBoxWithDate defendant={defendant} icon="calendar" />
+              </Box>
+              <Box component="section" marginBottom={5}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)